main.nf

```diff
@@ -18,10 +18,26 @@ nextflow.enable.dsl = 2
 
 WorkflowMain.initialise(workflow, params, log)
 
-include {METAGENOMICS_SR} from './workflows/metagenomics-sr'
+if (params.platform == 'illumina') {
+    include { ILLUMINA } from './workflows/illumina'
+} else if (params.platform == 'nanopore') {
+    include { NANOPORE } from './workflows/nanopore'
```

Comment:
should these end with `*.nf`? apologies if that's a dumb question

Reply:
I think either works and TBH I haven't been very consistent, so it a later cleaning up stage I will try to fix this, but it does work

Reply:
so far we have not been including and it's fine